dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESMappingUtilHelper.java

```diff
@@ -294,23 +295,28 @@ private void addMappingForRemainingFields(final Set<String> mappedFields,
     private void addMappingForContentTypeIfNeeded(final ContentType contentType,
             final Set<String> mappedFields, final String... indexes) {
         final Map<String, JSONObject> contentTypeMapping = new HashMap();
-        contentType.fields().stream().forEach(field-> {
-                try {
-                    addMappingForFieldIfNeeded(contentType, field,
-                            mappedFields, contentTypeMapping);
-                    putContentTypeMapping(contentType, contentTypeMapping, indexes);
-                } catch (Exception e) {
-                    handleInvalidCustomMappingError(
-                            "notification.reindexing.content.type.mapping.error",
-                            contentType.name(), indexes);
-                    final String message =
-                            "Error updating index mapping for content type " + contentType.name()
-                                    + ". This custom mapping will be ignored for index(es) " +
-                                    Arrays.stream(indexes).collect(Collectors.joining(","));
-                    Logger.warn(ESMappingUtilHelper.class, message, e);
+        try {
+            contentType.fields().stream().forEach(field-> {
```

Comment:
`stream()` method can be removed

Reply:
Why having nested try-catches if you are throwing a `DotRuntimeException` to be caught outside the loop?

Reply:
Because these exceptions need to be logged but never interrupt the whole process. This logic is part of a full reindex, which can take a lot of time in large datasets. Here, it is important to give more priority to the full process itself, because otherwise the application is useless, even when just setting a field mapping fails.

Reply:
By the way, the `stream()` suggestion was applied

Reply:
Fair enough! Thanks for clarifying.